spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/AdminController.java

```diff
@@ -166,8 +155,11 @@ public class AdminController {
     @Autowired
     GatewayHealthMonitorService gatewayHealthMonitorService;
 
+    @Autowired
+    OAuthUserDetailsService oAuthUserDetailsService;
+
     @RequestMapping(value = "/profiles", method = GET)
-    public ModelAndView goProfiles(Model model) throws IOException {
+    public ModelAndView goProfiles() throws IOException {
```

Comment:
I have not idea what this method is doing by reading its name. Could you please rename it?

Reply:
This code was not added by me, I think it was Paco who did it.
This change might have appeared on this PR, because I was trying to clean the unused code on the `AdminController`, and Intellij might have removed the `Model model`